README.md

```diff
@@ -1,13 +1,11 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px">
+        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="126px">
```

Comment:
In [package template](https://raw.githubusercontent.com/yiisoft/package-template/master/README.md) height is `100px`.
Why 126px ?

Reply:
I'll return it now. I looked in the config package, there is for some reason 126px :)